msgid ""
msgstr ""
"Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'bugs@pyload.org'\n"
"POT-Creation-Date: 2014-07-13 20:53+0200\n"
"PO-Revision-Date: 2015-01-18 18:30-0500\n"
"Last-Translator: pyloadTeam <team@pyload.org>\n"
"Language-Team: Russian\n"
"Language: ru_RU\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: crowdin.com\n"
"X-Crowdin-Project: pyload\n"
"X-Crowdin-File: /pyLoad/core.pot\n"

#: pyLoadCore.py:181
msgid "Received Quit signal"
msgstr "Получен сигнал для завершения работы"

#: pyLoadCore.py:302
#, python-format
msgid "pyLoad already running with pid %s"
msgstr "pyLoad уже запущен, pid %s"

#: pyLoadCore.py:316
#, python-format
msgid "Failed changing group: %s"
msgstr "Не удалось изменить группу: %s"

#: pyLoadCore.py:326
#, python-format
msgid "Failed changing user: %s"
msgstr "Не удалось изменить пользователя: %s"

#: pyLoadCore.py:328
msgid "folder for logs"
msgstr "папка отчетов"

#: pyLoadCore.py:339
msgid "Starting"
msgstr "Запускается"

#: pyLoadCore.py:340
#, python-format
msgid "Using home directory: %s"
msgstr "Основная папка: %s"

#: pyLoadCore.py:349
msgid "pycrypto to decode container files"
msgstr "pycrypto для декодирования файлов контейнеров"

#: pyLoadCore.py:352
msgid "folder for temporary files"
msgstr "папка временных файлов"

#: pyLoadCore.py:357
msgid "folder for downloads"
msgstr "папка для закачек"

#: pyLoadCore.py:360
msgid "OpenSSL for secure connection"
msgstr "OpenSSL для безопасного соединения"

#: pyLoadCore.py:364
msgid "Moving old user config to DB"
msgstr "Переместить прежнюю пользовательскую конфигурацию в БД"

#: pyLoadCore.py:367
msgid "Please check your logindata with ./pyLoadCore.py -u"
msgstr "Пожалуйста, проверьте свои учётные данные запустив ./pyLoadCore.py -u"

#: pyLoadCore.py:370
msgid "All links removed"
msgstr "Все ссылки удалены"

#: pyLoadCore.py:401
#, python-format
msgid "Downloadtime: %s"
msgstr "Время закачки: %s"

#: pyLoadCore.py:411
#, python-format
msgid "Free space: %s"
msgstr "Свободное место: %s"

#: pyLoadCore.py:431
msgid "Activating Accounts..."
msgstr "Активация аккаунта..."

#: pyLoadCore.py:437
msgid "Activating Plugins..."
msgstr "Активация плагина..."

#: pyLoadCore.py:440
msgid "pyLoad is up and running"
msgstr "pyLoad загружен и работает"

#: pyLoadCore.py:459
msgid "restarting pyLoad"
msgstr "перезапуск pyLoad"

#: pyLoadCore.py:463
msgid "pyLoad quits"
msgstr "завершение работы pyLoad"

#: pyLoadCore.py:520
#, python-format
msgid "Install %s"
msgstr "Установка %s"

#: pyLoadCore.py:556
#, python-format
msgid "could not find %(desc)s: %(name)s"
msgstr "не могу найти %(desc)s: %(name)s"

#: pyLoadCore.py:558
#, python-format
msgid "could not create %(desc)s: %(name)s"
msgstr "не могу создать %(desc)s: %(name)s"

#: pyLoadCore.py:579
msgid "shutting down..."
msgstr "завершение работы..."

#: pyLoadCore.py:596
msgid "error while shutting down"
msgstr "ошибка при завершении работы"

#: pyLoadCore.py:660
msgid "killed pyLoad from Terminal"
msgstr "pyLoad остановлен с терминала"

#: module/database/DatabaseBackend.py:174
msgid "Filedatabase was deleted due to incompatible version."
msgstr "Базы данных была удалена по причине несовместимой версии."

#: module/database/DatabaseBackend.py:189
msgid "Filedatabase could NOT be converted."
msgstr "База данных НЕ может быть преобразована."

#: module/database/DatabaseBackend.py:198
msgid "Database was converted from v2 to v3."
msgstr "База данных преобразована из v2 в v3."

#: module/database/DatabaseBackend.py:206
msgid "Database was converted from v3 to v4."
msgstr "База данных преобразована из v2 в v3."

#: module/database/DatabaseBackend.py:252
msgid "Converting old Django DB"
msgstr "Преобразование прежней БД Django"

#: module/database/FileDatabase.py:45
msgid "finished"
msgstr "закончено"

#: module/database/FileDatabase.py:45
msgid "offline"
msgstr "оф-лайн"

#: module/database/FileDatabase.py:45
msgid "online"
msgstr "он-лайн"

#: module/database/FileDatabase.py:45
msgid "queued"
msgstr "в очереди"

#: module/database/FileDatabase.py:45
msgid "skipped"
msgstr "пропущено"

#: module/database/FileDatabase.py:45
msgid "waiting"
msgstr "ожидание"

#: module/database/FileDatabase.py:45
msgid "temp. offline"
msgstr "временно недоступно"

#: module/database/FileDatabase.py:45
msgid "starting"
msgstr "запускается"

#: module/database/FileDatabase.py:45
msgid "failed"
msgstr "завершено с ошибкой"

#: module/database/FileDatabase.py:45
msgid "aborted"
msgstr "прервано"

#: module/database/FileDatabase.py:45
msgid "decrypting"
msgstr "раскодирование"

#: module/database/FileDatabase.py:45
msgid "custom"
msgstr "пользовательский"

#: module/database/FileDatabase.py:45
msgid "downloading"
msgstr "загружается"

#: module/database/FileDatabase.py:45
msgid "processing"
msgstr "обрабатывается"

#: module/database/FileDatabase.py:45
msgid "unknown"
msgstr "неизвестно"

#: module/database/FileDatabase.py:531 module/plugins/hooks/IRCInterface.py:74
#: module/plugins/hooks/XMPPInterface.py:83
#, python-format
msgid "Package finished: %s"
msgstr "Пакет завершен: %s"

#: module/remote/ThriftBackend.py:39
msgid "Using SSL ThriftBackend"
msgstr "Используется SSL ThriftBackend"

#: module/remote/RemoteManager.py:35
#, python-format
msgid "Remote backend error: %s"
msgstr "Ошибка удаленного сервиса: %s"

#: module/remote/RemoteManager.py:82
#, python-format
msgid "Starting %(name)s: %(addr)s:%(port)s"
msgstr "Запуск %(name)s: %(addr)s:%(port)s"

#: module/remote/RemoteManager.py:84
#, python-format
msgid "Failed loading backend %(name)s | %(error)s"
msgstr "Ошибка при загрузке %(name)s | %(error)s"

#: module/web/json_app.py:57
#, python-format
msgid "waiting %s"
msgstr "ожидание %s"

#: module/web/ServerThread.py:36
msgid "SSL certificates not found."
msgstr "SSL сертификаты не найдены."

#: module/web/ServerThread.py:40
#, python-format
msgid "Sorry, we dropped support for starting %s directly within pyLoad"
msgstr "К сожалению, поддержка в pyLoad прекращена начиная с версии %s"

#: module/web/ServerThread.py:41
msgid "You can use the threaded server which offers good performance and ssl,"
msgstr "Вы можете использовать threaded сервер для повышения производительности и использования SSL,"

#: module/web/ServerThread.py:42
#, python-format
msgid "of course you can still use your existing %s with pyLoads fastcgi server"
msgstr "но можно и использовать прежний сервер %s с pyLoads fastcgi"

#: module/web/ServerThread.py:43
msgid "sample configs are located in the module/web/servers directory"
msgstr "Примеры конфигурации web-серверов можно посмотреть в папке module/web/servers"

#: module/web/ServerThread.py:49
#, python-format
msgid "Can't use %(server)s, python-flup is not installed!"
msgstr "Использовать %(server)s невозможно, так как не установлен python-flup!"

#: module/web/ServerThread.py:58
#, python-format
msgid "Error importing lightweight server: %s"
msgstr "Ошибка импорта lightweight сервер:% с"

#: module/web/ServerThread.py:59
msgid "You need to download and compile bjoern, https://github.com/jonashaag/bjoern"
msgstr "Вам необходимо скачать и компилировать Bjoern, https://github.com/jonashaag/bjoern"

#: module/web/ServerThread.py:60
msgid "Copy the boern.so to module/lib folder or use setup.py install"
msgstr "Скопируйте boern.so в module/lib folder и не используйте setup.py для установки"

#: module/web/ServerThread.py:61
msgid "Of course you need to be familiar with linux and know how to compile software"
msgstr "Конечно, вы должны быть знакомы с Linux и знаете, как компилировать программы"

#: module/web/ServerThread.py:64
msgid "Server set to threaded, due to known performance problems on windows."
msgstr "Выбран сервер threaded, в связи с известными проблемами производительности на windows."

#: module/web/ServerThread.py:80 module/web/ServerThread.py:103
msgid "This server offers no SSL, please consider using threaded instead"
msgstr "Этот сервер не предоставляет возможность использовать SSL, пожалуйста рассмотрите возможность использовать сервер threaded взамен выбранного"

#: module/web/ServerThread.py:82
#, python-format
msgid "Starting builtin webserver: %(host)s:%(port)d"
msgstr "Запуск встроенного веб-сервера django: %(host)s:%(port)d"

#: module/web/ServerThread.py:87
#, python-format
msgid "Starting threaded SSL webserver: %(host)s:%(port)d"
msgstr "Запуск веб-сервера lighttpd: %(host)s:%(port)d"

#: module/web/ServerThread.py:91
#, python-format
msgid "Starting threaded webserver: %(host)s:%(port)d"
msgstr "Запуск threaded веб-сервера: %(host)s:%(port)d"

#: module/web/ServerThread.py:97
#, python-format
msgid "Starting fastcgi server: %(host)s:%(port)d"
msgstr "Запуск fastcgi-сервера: %(host)s:%(port)d"

#: module/web/ServerThread.py:105
#, python-format
msgid "Starting lightweight webserver (bjoern): %(host)s:%(port)d"
msgstr "Запуск lightweight веб-сервер (Bjoern):% (host)s:% (port)d"

#: module/web/pyload_app.py:127
msgid "You dont have permission to access this page."
msgstr "У вас нет прав на просмотр страницы."

#: module/web/pyload_app.py:195
msgid "Download directory not found."
msgstr "Папка для закачек не найдена."

#: module/web/pyload_app.py:262 module/web/pyload_app.py:269
msgid "unlimited"
msgstr "неограниченный"

#: module/web/pyload_app.py:264 module/web/pyload_app.py:271
msgid "not available"
msgstr "не доступно"

#: module/web/pyload_app.py:509
msgid "Run pyLoadCore.py -s to access the setup."
msgstr "Для доступа к настройкам запустите pyLoadCore.py -s."

#: module/network/HTTPDownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Загрузка параллельных потоков данных завершена с ошибками, возврат к однопоточной загрузке | %s"

#: module/PluginThread.py:183
#, python-format
msgid "Download starts: %s"
msgstr "Загрузка начата: %s"

#: module/PluginThread.py:189
#, python-format
msgid "Download finished: %s"
msgstr "Загрузка завершена: %s"

#: module/PluginThread.py:194 module/PluginThread.py:365
#, python-format
msgid "Plugin %s is missing a function."
msgstr "В плагине %s нет функции."

#: module/PluginThread.py:202 module/PluginThread.py:265
#: module/PluginThread.py:382
#, python-format
msgid "Download aborted: %s"
msgstr "Загрузка прервана: %s"

#: module/PluginThread.py:222
#, python-format
msgid "Download restarted: %(name)s | %(msg)s"
msgstr "Ошибка загрузки: %(name)s | %(msg)s"

#: module/PluginThread.py:231 module/PluginThread.py:373
#, python-format
msgid "Download is offline: %s"
msgstr "Загрузка недоступна: %s"

#: module/PluginThread.py:234
#, python-format
msgid "Download is temporary offline: %s"
msgstr "Загрузка временно недоступна: %s"

#: module/PluginThread.py:237 module/PluginThread.py:304
#, python-format
msgid "Download failed: %(name)s | %(msg)s"
msgstr "Ошибка загрузки: %(name)s | %(msg)s"

#: module/PluginThread.py:254
msgid "Couldn't connect to host or connection reset, waiting 1 minute and retry."
msgstr "Невозможно соединиться с провайдером или срыв соединения, следующая попытка через одну минуту."

#: module/PluginThread.py:290
#, python-format
msgid "Download skipped: %(name)s due to %(plugin)s"
msgstr "Загрузка пропущена: %(name)s @ %(plugin)s"

#: module/PluginThread.py:361
#, python-format
msgid "Decrypting starts: %s"
msgstr "Декодирование начато: %s"

#: module/PluginThread.py:376 module/PluginThread.py:394
#, python-format
msgid "Decrypting failed: %(name)s | %(msg)s"
msgstr "Ошибка декодирования: %(name)s | %(msg)s"

#: module/PluginThread.py:388
#, python-format
msgid "Retrying %s"
msgstr "Повторная попытка %s"

#: module/PluginThread.py:635
#, python-format
msgid "Info Fetching for %(name)s failed | %(err)s"
msgstr "Получение информации по %(name)s завершилось ошибкой | %(err)s"

#: module/HookManager.py:90 module/plugins/Hook.py:103
#, python-format
msgid "Error executing hooks: %s"
msgstr "Ощибка выполнения захватов: %s"

#: module/HookManager.py:140
#, python-format
msgid "Failed activating %(name)s"
msgstr "Ошибка активации %(name)s"

#: module/HookManager.py:144
#, python-format
msgid "Activated plugins: %s"
msgstr "Активированные плагины:% S"

#: module/HookManager.py:145
#, python-format
msgid "Deactivate plugins: %s"
msgstr "Отключить плагины:%s"

#: module/ThreadManager.py:137
#, python-format
msgid "Reconnect Failed: %s"
msgstr "Переподключение не удалось: %s"

#: module/ThreadManager.py:176
msgid "Reconnect script not found!"
msgstr "Скрипт переподключения не найден!"

#: module/ThreadManager.py:182
msgid "Starting reconnect"
msgstr "Начинаем переподключение"

#: module/ThreadManager.py:196
msgid "Failed executing reconnect script!"
msgstr "Ошибка при выполнении скрипта переподключения"

#: module/ThreadManager.py:208
#, python-format
msgid "Reconnected, new IP: %s"
msgstr "Переподключено, новый IP: %s"

#: module/ThreadManager.py:288
msgid "Not enough space left on device"
msgstr "Не хватает места на устройстве."

#: module/plugins/Account.py:85 module/plugins/Account.py:90
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "Не могу войти с учетной записью %(user)s | %(msg)s"

#: module/plugins/Account.py:86
msgid "Wrong Password"
msgstr "Неправильный пароль"

#: module/plugins/Account.py:243
#, python-format
msgid "Your Time %s has wrong format, use: 1:22-3:44"
msgstr "Время %s имеет неправильный формат, используйте формат 1:22-3:44"

#: module/plugins/Account.py:269
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "Недостаточно траффика в аккаунте %s, следующая проверка через 30 минут"

#: module/plugins/Account.py:276
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "Истек срок действия аккаунта %s, следующая проверка через 1 час"

#: module/plugins/crypter/SerienjunkiesOrg.py:126
msgid "Downloadlimit reached"
msgstr "Достигнут лимит загрузок"

#: module/plugins/PluginManager.py:153
#, python-format
msgid "%s has a invalid pattern."
msgstr "%s имеет не правильный шаблон."

#: module/plugins/PluginManager.py:272
#, python-format
msgid "Error importing %(name)s: %(msg)s"
msgstr "Ошибка импорта %(name)s: %(msg)s"

#: module/plugins/internal/MultiHoster.py:132
msgid "No Hoster loaded"
msgstr "Не загружен ни один из файлобменников"

#: module/plugins/accounts/BitshareCom.py:38
msgid "Activate direct Download in your Bitshare Account"
msgstr "В своей учётной записи включите опцию 'direct Download'"

#: module/plugins/container/LinkList.py:64
msgid "LinkList could not be cleared."
msgstr "Список ссылок не может быть очищен."

#: module/plugins/AccountManager.py:88
msgid "Account settings deleted, due to new config format."
msgstr "Установки учетной записи удалены в связи с новым форматом."

#: module/plugins/hoster/BasePlugin.py:65
msgid "Authorization required (username:password)"
msgstr "Необходима авторизация (имя пользователя:пароль)"

#: module/plugins/hoster/SimplydebridCom.py:23
#: module/plugins/hoster/RealdebridCom.py:40
#: module/plugins/hoster/FreeWayMe.py:39 module/plugins/hoster/ZeveraCom.py:21
#: module/plugins/hoster/UnrestrictLi.py:52
#: module/plugins/hoster/Vipleech4uCom.py:30
#: module/plugins/hoster/Premium4Me.py:27 module/plugins/hoster/FastixRu.py:36
#: module/plugins/hoster/SimplyPremiumCom.py:52
#: module/plugins/hoster/MegaDebridEu.py:46
#: module/plugins/hoster/AlldebridCom.py:38
#: module/plugins/hoster/LinksnappyCom.py:29
#: module/plugins/hoster/DebridItaliaCom.py:39
#: module/plugins/hoster/RPNetBiz.py:28
#: module/plugins/hoster/MultiDebridCom.py:40
#: module/plugins/hoster/ReloadCc.py:26 module/plugins/hoster/OverLoadMe.py:38
#: module/plugins/hoster/RehostTo.py:25
#: module/plugins/hoster/PremiumizeMe.py:24
#: module/plugins/hooks/RPNetBiz.py:45
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Пожалуйста, введите ваш аккаунт для %s или отключить этот плагин"

#: module/plugins/hoster/FilesMailRu.py:98
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Найден HTML-код в файле загрузки (%s)... ошибка перенаправления? Загрузка будет перезапущена."

#: module/plugins/hoster/NetloadIn.py:145
#: module/plugins/hoster/NetloadIn.py:169
msgid "File temporarily not available"
msgstr "Файл временно недоступен"

#: module/plugins/hoster/NetloadIn.py:182
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: ожидание между загрузками %d сек."

#: module/plugins/hoster/NetloadIn.py:213
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: ожидание картинки %d сек."

#: module/plugins/hoster/NetloadIn.py:251
msgid "Downloaded File was empty"
msgstr "Загруженный файл пуст"

#: module/plugins/hoster/MegaDebridEu.py:49
#, python-format
msgid "Impossible to connect to %s"
msgstr "Невозможно подключиться к %s"

#: module/plugins/hoster/MegaDebridEu.py:89
#, python-format
msgid "Impossible to debrid %s"
msgstr "Невозможно раскрутить ссылку %s"

#: module/plugins/hoster/UploadedTo.py:129
msgid "API key invalid"
msgstr "Неверный ключ API"

#: module/plugins/hoster/UploadedTo.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: Осталось недостаточно трафика"

#: module/plugins/hoster/UploadedTo.py:156
msgid "Traffic exceeded"
msgstr "Трафик закончился"

#: module/plugins/hoster/RapidshareCom.py:99
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Traffic Share (прямая загрузка)"

#: module/plugins/hoster/RapidshareCom.py:126
#: module/plugins/hoster/RapidshareCom.py:193
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Уже идет загрузка с этого IP, ждем 60 секунд"

#: module/plugins/hoster/RapidshareCom.py:130
msgid "Invalid Auth Code, download will be restarted"
msgstr "Неправильный код идентификации, загрузка будет перезапущена"

#: module/plugins/hoster/RapidshareCom.py:198
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: Нет свободного слота"

#: module/plugins/hoster/RapidshareCom.py:201
msgid "You need a premium account for this file"
msgstr "Для этого файла нужен премиум аккаунт"

#: module/plugins/hoster/RapidshareCom.py:203
msgid "Filename reported invalid"
msgstr "Имя файла указано неверно"

#: module/plugins/hoster/FileserveCom.py:100
msgid "Parallel download error, now waiting 60s."
msgstr "Ошибка параллельной загрузки, ждем 60 секунд."

#: module/plugins/hoster/FileserveCom.py:216
msgid "Not logged in."
msgstr "Вход не произведен."

#: module/plugins/hoster/MegaNz.py:56
msgid "Decryption failed"
msgstr "Сбой расшифровки"

#: module/plugins/hoster/MegaNz.py:106
msgid "No file key provided in the URL"
msgstr "Нет ключа файла в URL-адресе"

#: module/plugins/hoster/MegaNz.py:118
msgid "Error code:"
msgstr "Код ошибки:"

#: module/plugins/Container.py:68
msgid "File not exists."
msgstr "Файл не существует."

#: module/plugins/hooks/UpdateManager.py:97
msgid "Not able to connect server to get updates"
msgstr "Не удалось подключиться к серверу для получения обновлений"

#: module/plugins/hooks/UpdateManager.py:111
msgid "No pyLoad version available"
msgstr "Нет доступных версий pyLoad"

#: module/plugins/hooks/UpdateManager.py:118
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  Доступна новая версия pyLoad %s  ***"

#: module/plugins/hooks/UpdateManager.py:119
msgid "***  Get it here: https://github.com/pyload/pyload/releases  ***"
msgstr "*** Получить здесь: https://github.com/pyload/pyload/releases ***"

#: module/plugins/hooks/UpdateManager.py:170
#, python-format
msgid "New version of [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"
msgstr "Новая версия [%(type)s] %(name)s (v%(oldver)s -> v%(newver)s)"

#: module/plugins/hooks/UpdateManager.py:180
#: module/plugins/hooks/UpdateManager.py:185
#, python-format
msgid "Error when updating plugin %s"
msgstr "Ошибка при обновлении плагина %s"

#: module/plugins/hooks/UpdateManager.py:185
msgid "Version mismatch"
msgstr "Конфликт версии"

#: module/plugins/hooks/UpdateManager.py:196
#, python-format
msgid "Removed blacklisted plugin: [%(type)s] %(name)s"
msgstr "Заблокированный плагин удалён: [%(type)s] %(name)s"

#: module/plugins/hooks/UpdateManager.py:204
msgid "Plugins updated and reloaded"
msgstr "Дополнения обновлены и перезагружены"

#: module/plugins/hooks/UpdateManager.py:206
msgid "*** Plugins have been updated, pyLoad will be restarted now ***"
msgstr "*** Плагины были обновлены, pyLoad будет перезапущен ***"

#: module/plugins/hooks/UpdateManager.py:211
msgid "No plugin updates available"
msgstr "Нет обновлений для дополнений"

#: module/plugins/hooks/IRCInterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Загрузка завершена: %(name)s @ %(plugin)s "

#: module/plugins/hooks/IRCInterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Новый запрос картинки: %s"

#: module/plugins/hooks/IRCInterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Ответ с текстом на картинке %s"

#: module/plugins/hooks/Premium4Me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Пожалуйста, сначала добавьте учетную запись premium.to и перезапустите pyLoad"

#: module/plugins/hooks/HotFolder.py:81
#, python-format
msgid "Added %s from HotFolder"
msgstr "Добавлен %s из HotFolder"

#: module/plugins/hooks/ExtractArchive.py:94
#, python-format
msgid "No %s installed"
msgstr "%s не установлен"

#: module/plugins/hooks/ExtractArchive.py:96
#: module/plugins/hooks/ExtractArchive.py:101
#, python-format
msgid "Could not activate %s"
msgstr "Не возможно активировать %s"

#: module/plugins/hooks/ExtractArchive.py:106
msgid "Activated"
msgstr "Активирован(a/о)"

#: module/plugins/hooks/ExtractArchive.py:108
msgid "No Extract plugins activated"
msgstr "Не активировано дополнений для извлечения/распаковки"

#: module/plugins/hooks/ExtractArchive.py:120
#, python-format
msgid "Package %s queued for later extracting"
msgstr "Пакет %s поставлен в очередь для последующего извлечения/распаковки"

#: module/plugins/hooks/ExtractArchive.py:143
#, python-format
msgid "Check package %s"
msgstr "Проверка пакета %s"

#: module/plugins/hooks/ExtractArchive.py:184
#, python-format
msgid "Extract to %s"
msgstr "Извлечение/распаковка в %s"

#: module/plugins/hooks/ExtractArchive.py:199
msgid "No files found to extract"
msgstr "Не найдено файлов для извлечения"

#: module/plugins/hooks/ExtractArchive.py:206
msgid "extracting"
msgstr "извлекается/распаковывается"

#: module/plugins/hooks/ExtractArchive.py:217
msgid "Password protected"
msgstr "Защищено паролем"

#: module/plugins/hooks/ExtractArchive.py:238
msgid "Wrong password"
msgstr "Неправильный пароль"

#: module/plugins/hooks/ExtractArchive.py:246
#, python-format
msgid "Deleting %s files"
msgstr "Удаление файлов %s"

#: module/plugins/hooks/ExtractArchive.py:253
msgid "Extracting finished"
msgstr "Извлечениe/распаковка завершено(a)"

#: module/plugins/hooks/ExtractArchive.py:259
msgid "Archive Error"
msgstr "Ошибка архива"

#: module/plugins/hooks/ExtractArchive.py:261
msgid "CRC Mismatch"
msgstr "Ошибка контрольной суммы"

#: module/plugins/hooks/ExtractArchive.py:265
msgid "Unknown Error"
msgstr "Неизвестная ошибка"

#: module/plugins/hooks/ExtractArchive.py:317
msgid "Setting User and Group failed"
msgstr "Ошибка установки пользователя и группы: %s"

#: module/plugins/hooks/ClickAndLoad.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: Порт 9666 уже занят"

#: module/plugins/hooks/CaptchaTrader.py:70
#: module/plugins/hooks/Captcha9kw.py:60
#: module/plugins/hooks/ExpertDecoders.py:51
#, python-format
msgid "%s credits left"
msgstr "Осталось %s кредитов"

#: module/plugins/hooks/CaptchaTrader.py:118
msgid "Could not send response."
msgstr "Не удалось отправить ответ."

#: module/plugins/hooks/CaptchaTrader.py:136
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "Недостаточно средств на вашем аккаунте CaptchaTrader"

#: module/plugins/hooks/LinkdecrypterCom.py:45
msgid "Crypter list not found"
msgstr "Список шифрования не найден"

#: module/plugins/hooks/LinkdecrypterCom.py:59
msgid "Crypter list is empty"
msgstr "Список шифрования пуст"

#: module/plugins/hooks/XMPPInterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Загрузка завершена: %(name)s @ %(plugin)s"

#: module/plugins/hooks/Captcha9kw.py:94
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Новая(ые) CaptchaID из загрузки: %s: %s"

#: module/plugins/hooks/Captcha9kw.py:130
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "Ваша учетная запись 9kw.eu имеет не достаточно кредитов"

#: module/plugins/hooks/ExternalScripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Установленные скрипты для %s: "

#: module/plugins/hooks/ExternalScripts.py:69
msgid "Script not executable:"
msgstr "Не возможно запустить скрипт:"

#: module/plugins/hooks/ExternalScripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Ошибка в %(script): %(error)s"

#: module/plugins/hooks/ExpertDecoders.py:96
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "Учетная запись ExpertDecoders не имеет достаточно кредитов"

#: module/plugins/hooks/RehostTo.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Пожалуйста сначала добавьте ваш rehost.to аккаунт и перезапустите pyLoad"

#: module/plugins/hooks/PremiumizeMe.py:50
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Пожалуйста сначала добавьте действительную учетную запись для premiumize.me и перезапустить pyLoad."

#: module/plugins/hooks/CaptchaBrotherhood.py:70
#, python-format
msgid "%d credits left"
msgstr "Осталось %s кредитов"

#: module/plugins/Plugin.py:389
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil и tesseract не установлены. Не подключен клиент для расшифровки картинок"

#: module/plugins/Plugin.py:393
msgid "No captcha result obtained in appropiate time by any of the plugins."
msgstr "Не получен результат расшифровки картинки в течение отведённого времени ни одним из дополнений"

#: module/plugins/Plugin.py:498 module/plugins/Plugin.py:532
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Ошибка установки пользователя и группы: %s"

#: module/CaptchaManager.py:78
msgid "No Client connected for captcha decrypting"
msgstr "Не подключен клиент для расшифровки картинок"

#: module/Api.py:330
#, python-format
msgid "Added package %(name)s containing %(count)d links"
msgstr "Добавлен контейнер %(name)s с %(count)d сылками"

#: module/Api.py:593
#, python-format
msgid "Added %(count)d links to package #%(package)d "
msgstr "Добавлено %(count)d ссылок в контейнер #%(package)d "

#: module/common/JsEngine.py:156
msgid "No js engine detected, please install either Spidermonkey, ossp-js, pyv8 or rhino"
msgstr "Не обнаружена js платформа, пожалуйста установите либо Spidermonkey, ossp-js, pyv8 или rhino"

